website/common/script/ops/changeClass.js

```diff
@@ -44,26 +43,6 @@ module.exports = function changeClass (user, req = {}, analytics) {
     user.flags.classSelected = true;
 
     each(['weapon', 'armor', 'shield', 'head'], (type) => {
```

Comment:
`armor` and `head` can be removed from this `each`

It might even be clearer to get rid of the `each` and just have lines like below, but I'm not certain this is better so disagreements are welcome. :)

    user.items.gear.owned[`weapon_${klass}_0`] = true;
    if (klass === 'rogue')  user.items.gear.owned[`shield_${klass}_0`] = true;